server/binding.js

```diff
@@ -359,17 +363,23 @@ module.exports = {
    * @returns {Array} array of object of each wiggle file
    */
   listBindingGenes: function(wiggleAddr) {
```

Comment:
Looks like argument "wiggleAddr" is never used. Addrs in other listing functions are also not used. Maybe create a listing files utils function for all listing queries, and pass in different folder arguments.